src/components/VContentSwitcher/VSearchTypePopover.vue

```diff
@@ -67,6 +67,7 @@ export default defineComponent({
     /**
      * When in the searchbar, content switcher button has a border when the
      * search bar group is hovered on.
+     *
      * @type {ComputedRef<boolean>}
```

Comment:
I guess this type annotation is incorrect because it did not import `ComputedRef` from `@nuxt/composition-api`... WebStorm automatically inserts types without the imports.